iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/writer/WriterBuilder.java

```diff
@@ -120,11 +121,20 @@ public HiveIcebergWriter build() {
         new HiveFileWriterFactory(table, dataFileFormat, dataSchema, null, deleteFileFormat, null, null, null,
             skipRowData ? null : dataSchema);
 
+    boolean copyOnWriteMode = RowLevelOperationMode.COPY_ON_WRITE.modeName().equalsIgnoreCase(
+        properties.get(TableProperties.DELETE_MODE)) &&
+        operation == Operation.DELETE;
```

Comment:
Can't we use one line? Is this line longer than 120 characters?
```suggestion
        properties.get(TableProperties.DELETE_MODE)) && operation == Operation.DELETE;
```

Reply:
changed